src/main/io/osd.c

```diff
@@ -3216,7 +3209,6 @@ PG_RESET_TEMPLATE(osdConfig_t, osdConfig,
     .hud_radar_disp = SETTING_OSD_HUD_RADAR_DISP_DEFAULT,
     .hud_radar_range_min = SETTING_OSD_HUD_RADAR_RANGE_MIN_DEFAULT,
     .hud_radar_range_max = SETTING_OSD_HUD_RADAR_RANGE_MAX_DEFAULT,
-    .hud_radar_nearest = SETTING_OSD_HUD_RADAR_NEAREST_DEFAULT,
```

Comment:
We will need a version bump for `PG_OSD_CONFIG` when config structure changes

Reply:
Ok, this one ?
![Capture d’écran 2021-11-24 125647](https://user-images.githubusercontent.com/25566932/143234202-a2f8e1ed-677f-4d43-846f-be58306c1b6b.jpg)